Exam28/Exam28/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exam28
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int maxNum = Convert.ToInt32("11111111111111111111111111111111", 2);
+            int num = 0;
+            int summ = 0;
+            while (num >= maxNum)
+            {
+                summ = summ + L(IntToS(num));
+                num++;
+            }
```

Comment:
А здесь, конечно, суммировать значения L(s) для всех не меньших чем `maxNum`:
```suggestion
            uint num = 0;
            int summ = 0;
            while (num <= maxNum)
            {
                summ += L(IntToS(num));
                num++;
            }
```